cmd/flarectl/flarectl.go

```diff
@@ -378,7 +427,7 @@ func main() {
 	app := cli.NewApp()
 	app.Name = "flarectl"
 	app.Usage = "CloudFlare CLI"
-	app.Version = "2015.12.0"
+	app.Version = "2016.4.0"
```

Comment:
I think git tags should work. We can start with version 1.0.0 a bunch of the current structural changes are done and the library is more feature complete. In my eyes versioning is also as much about having a changelog so users of the library have a way of identifying any changes they need to be aware of, without having to look through commit histories and such.